devices/common_patches/g0_usb_enum.yaml

```diff
@@ -0,0 +1,21 @@
+# Fixes duplicate enum values
+USB:
+  CNTR:
+    USBRST:
+      _replace_enum:
+        B_0x0: [0, "No effect"]
```

Comment:
In such form enums are useful. Give them meaningful name.

Reply:
Thanks, should be better now.